### DISCUSSION TITLE
Opinions on a new and very similar framework Gravity

### DISCUSSION THREAD

**Discussion by PointSingularity** [trpc/trpc#1890] [General]
Recently a new RPC framework has been released called [Gravity](https://gravity.digitak.dev).

I don't know if anyone used it already, but my question is what your opinions are about it compared to tRPC?

**Comment by sachinraja** [comment]
One major difference I see: Gravity generates functions (with validation) based off your types, while validation frameworks like Zod are used in tRPC. Note that tRPC does not rely on any sort of generation. They do have [a comparison page](https://gravity-blond.vercel.app/docs/comparisons/tRPC) but I don't think it's correct. I've made a PR to fix it https://github.com/digital-loukoum/gravity/pull/6.

**Reply by PointSingularity** [reply]
Thanks for the answer!
Yeah, the docs aren't complete, for example tRPC is platform-agnostic just like Gravity even though Gravity docs say otherwise.
